docs/developer/filtering.mdx

```diff
@@ -0,0 +1,311 @@
+---
+title: Filtering
+---
+
+:::info
+
+The "where" filtering approach was introduced in **Saleor 3.11**. For now, it's only
+available on the `Attribute` type.
```

Comment:
```suggestion
The `where` filtering approach was introduced in **Saleor 3.11**. For now, it's only
available on the `Attribute` type.
```

Also, regarding that:
>  For now, it's only available on the `Attribute` type.

Isn't it the [`attributes`](https://docs.saleor.io/docs/3.x/api-reference/queries/attributes) query?

Reply:
Like I mean here that it's done for `Attributes` type with reference to the whole idea of adding the filtering input for each `Attribute` field, and adding a new filter on `attributes` query.